src/lib/misc.cpp

```diff
@@ -86,6 +86,12 @@ __RCSID("$NetBSD: misc.c,v 1.41 2012/03/05 02:20:18 christos Exp $");
 #include "utils.h"
 #include "json_utils.h"
 
+#ifdef _WIN32
+#include <windows.h>
+#include <locale>
+#include <codecvt>
+#endif
+
 #ifdef WIN32
```

Comment:
Just curious - how WIN32 differs from _WIN32?

Reply:
`_WIN32` is defined by the compiler, `WIN32` is sort of overridable by user.
I use `_WIN32` everywhere, this `WIN32` is leftover from some other developer.
So, given that we're not planning to re-define it, it's all the same. I'll make it `_WIN32` for consistency.